server/server.go

```diff
@@ -42,8 +42,9 @@ func NewConfiguredServer() *Server {
 		InterruptChan:  make(chan os.Signal),
 		DefaultHeaders: make(map[string]string),
 	}
-	s.DefaultHeaders["X-Access-Control-Allow-Origin"] = "*"
-	s.DefaultHeaders["Server"] = "lbry.tv"
+	s.DefaultHeaders["Access-Control-Allow-Origin"] = "*"
+	s.DefaultHeaders["Content-Type"] = "application/json; charset=utf-8"
```

Comment:
Unfortunately cannot set server-wide content-type to `application/json` — we are also serving video and plain html here. I'll fix it for `/api/proxy` endpoint though.

Reply:
Ah. Right.